src/activation.jl

```diff
@@ -225,8 +225,40 @@ See [Softshrink Activation Function](https://www.gabormelli.com/RKB/Softshrink_A
 """
 softshrink(x::Real, λ = oftype(x/1, 0.5)) = min(max(zero(x), x - λ), x + λ)
 
+"""
+    ptanh(x)
+
+Piecewise Approximation of Tanh Function
+
+See [Activation Functions](https://www.gabormelli.com/RKB/Softshrink_Activation_Function).
```

Comment:
I think you have the wrong link here. 

And I also think the docstring should explain how good/bad the approximation is, and ideally whether its gradient is continuous, etc.